tests/torch/test_brevitas_qat.py

```diff
@@ -542,7 +542,7 @@ def test_brevitas_power_of_two(
             elif manual_rounding:
                 # If manual rounding was set, LSBs_to_remove must be equal
                 # to the accumulator size minus the requested rounding_threshold_bits
-                assert node_op.rounding_threshold_bits == manual_rounding
+                assert node_op.rounding_threshold_bits.get("n_bits", None) == manual_rounding
```

Comment:
this can fail if `rounding_threshold_bits` is an int right ?

Reply:
@jfrery ?

Reply:
Yes right.